src/Utilities/Request.php

```diff
@@ -86,7 +86,7 @@ public function orderableColumns(): array
             /** @var string $direction */
             $direction = $this->request->input("order.$i.dir");
```

Comment:
Good catch @mgralikowski, maybe we can just set the default value here instead?

```suggestion
            $direction = $this->request->input("order.$i.dir", 'desc');
```

Reply:
I think it's ok since ensuring 'asc' or 'desc' is a security protection for SQL injection attacks.

Or we can fix the docblock instead to `/** @var string|null $direction */`

Reply:
Docblock would not resolve the PHP error ;)

IMHO - my proposal is more clear, because in so, there is only a single place where you define a default sorting - `desc`, no matter whether the passed value is undefined (null), incorrect, or is an attempt of SQL injection  ;) 

Another suggestion if you don't like this one is cast input to string:

`(string) $this->request->input("order.$i.dir");`

Reply:
> Docblock would not resolve the PHP error ;)

Oh, I was thinking about the static analysis since it's currently set as a string but the value can be null which is why you came up with the proposal to check if the `$direction` is set. ^_^

Reply:
Anyway, this is good to go.